bindings/matrix-sdk-crypto-nodejs/src/machine.rs

```diff
@@ -389,8 +386,7 @@ impl OlmMachine {
         event: String,
         room_id: &identifiers::RoomId,
     ) -> napi::Result<responses::DecryptedRoomEvent> {
-        let event: OriginalSyncRoomEncryptedEvent =
-            serde_json::from_str(event.as_str()).map_err(into_err)?;
+        let event = Raw::from_json(RawValue::from_string(event.to_string()).map_err(into_err)?);
```

Comment:
Same thing.

Reply:
Oh wait, I copy pasted the lines from the previous crates, but here we do have an owned string. So by your argument we should use `RawValue::from_string()` here?

Reply:
Yeah, here it could actually be beneficial. I'll also add `from_json_string` to `Raw` so this becomes one function call instead of two.